erpnext/regional/india/__init__.py

```diff
@@ -79,4 +79,44 @@
  "Uttar Pradesh": "09",
  "Uttarakhand": "05",
  "West Bengal": "19",
+}
+
+number_state_mapping = {
```

Comment:
Do we really need it? We can use reverse mapping of state_numbers.
`inv_map = {v: k for k, v in my_map.iteritems()}`